lib/puppet/trusted_external.rb

```diff
@@ -3,11 +3,35 @@ module Puppet::TrustedExternal
   def retrieve(certname)
     command = Puppet[:trusted_external_command]
```

Comment:
Recommend adding `:type => :file_or_directory` in defaults.rb so it's clear that the setting can be either a file or directory: https://github.com/puppetlabs/puppet/blob/1b5210a3920fe5430201ba185b1247ffd76166e3/lib/puppet/defaults.rb#L591

Reply:
Thanks, updated.